.env.example

```
@@ -11,9 +11,15 @@ SITE_URL=http://127.0.0.1
 # Expose the UI on this port
 UI_PORT=3000
 
+# If UI is on a different host
+#UI_HOST
```

Variables should have example values
```suggestion
#UI_HOST=127.0.0.1
```